modules/kinesis/src/it/resources/enrich/stop.hocon

```diff
@@ -0,0 +1,40 @@
+{
+  "input": {
+    "type": "Kinesis"
+    "appName": "enrich-kinesis-it-stop"
```

Comment:
I was wondering how adding new tests cases in the future would look like. If I want to add different scenario for my instance of enrich I need to make sure `appName` is unique across all tests? If that's true... how painful is that? 

What options do we have: 

1. New localstack + enrich per test. Everything is isolated, we don't have to worry about anything. Problem: it can take a lot of time. Out of curiosity, how much time does it take to run localstack container? And enrich? 
2. Shared localstack + enrich per test. We need to remember about unique app name. 

What about...generating `appName` dynamically like: `enrich-${UUID}` and setting it using JVM props like: `-D.input.appName` in `GenericContainer` ? Does it make sense or is it overkill?

Reply:
> What about... generating each appName randomly ?

Although feasible, I don't think that this is worth the effort, as it requires to template the config files, and update them with the random UUID before mounting them in the container.

I think that putting the test name in the app name is straightforward and good enough. We can revisit that if we suddenly start to add many tests (which will not happen in some near future).

Reply:
And this: 

>setting it using JVM props like: -D.input.appName

So no appName in the config file, it would be passed by JVM props. No need to template anything then I think?

Reply:
I don't get it, having a random UUID is only useful if we have one random UUID *per appName*, how having one for all the tests would help ?

Reply:
I mean set JVM props here: 

```
    val container = GenericContainer(
      dockerImage = "snowplow/snowplow-enrich-kinesis:latest",
      env =
        Map(
          "AWS_REGION" -> "eu-central-1",
          "AWS_ACCESS_KEY_ID" -> "foo",
          "AWS_SECRET_ACCESS_KEY" -> "bar",
         "JVM_HOCON_CONFIG_HERE"
        ),

```

when creating enrich container. Would that work?

Reply:
Or...delete DynamoDB tables after each test case?

Reply:
> Or...delete DynamoDB tables after each test case?

No, as we want the tests to run in parallel to save time.

> I mean set JVM props here

Ah right, I get it! It might be possible, let me try.